java/test/src/main/java/io/ray/test/JobConfigTest.java

```diff
@@ -33,10 +33,6 @@ public void testJvmOptions() {
     Assert.assertEquals("998", Ray.task(JobConfigTest::getJvmOptions, "Y").remote().get());
   }
 
-  public void testNumJavaWorkersPerProcess() {
-    Assert.assertEquals(TestUtils.getNumWorkersPerProcess(), 3);
```

Comment:
Delete `System.setProperty("ray.job.num-java-workers-per-process", "3");` please.

You need to search and delete `num-java-workers-per-process`, `num_java_workers_per_process`, `numJavaWorkersPerProcess`.

Reply:
Good catch.